php-classes/Slate/CBL/Demonstrations/DemonstrationSkillsRequestHandler.php

```diff
@@ -29,8 +29,15 @@ public static function handleBrowseRequest($options = [], $conditions = [], $res
                 return static::throwNotFoundError('Student not found');
             }
 
-            if (!$GLOBALS['Session']->hasAccountLevel('Staff') && $Student->ID != $GLOBALS['Session']->PersonID) {
-                return static::throwUnauthorizedError('Only staff may browse others\' records');
+            if (
+                    !$GLOBALS['Session']->hasAccountLevel('Staff') &&
+                    $Student->ID != $GLOBALS['Session']->PersonID &&
+                    !$GuardianRelationship = \Emergence\People\GuardianRelationship::getByWhere([
+                        'PersonID' => $GLOBALS['Session']->PersonID,
+                        'RelatedPersonID' => $Student->ID
+                    ]))
+                {
```

Comment:
@nbey drop the closing paren for the if to its own line, align it with the opening `if`, and then put the opening curly bracket after it like you see here: http://stackoverflow.com/questions/23758916/psr-2-standard-for-long-if-conditions